src/QueueController.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace UrbanIndo\Yii2\Queue;
+
+/**
+ * QueueController is a web controller to post job via url.
+ *
+ * @author adinata
+ */
+class QueueController extends \yii\web\Controller{
+    public $enableCsrfValidation = false;
+    public $queueComponent = 'queue';
+    
+    /**
+     * 
+     * @return type
+     */
+    public function actionPost() {
+        $route = \Yii::$app->getRequest()->post('route');
+        $data = \Yii::$app->getRequest()->post('data');
+        if (is_string($data)) {
+            $data = \yii\helpers\Json::decode($data);
+        } else {
```

Comment:
Remove the else. If the app accepts application/json POST, this will automatically be an object. Just in case it's traditional POST the data is in string.